### PR TITLE
Please add I-Mobile support to amp-ad

### PR DESCRIPTION
**Pull request by imobile**
Please add support to amp-ad for the imobile type.
Our CLA: i-mobile Co.,Ltd

Looking forward to hearing from your response!

### CODE REVIEW

**Comment by imobile** [comment]
Hello we signed the CLA as a corporation, would you please have it double checked?
Our CLA: i-mobile Co.,Ltd

**Comment by imobile** [comment]
Thank you very much for confirming! Looking forward to your feedbacks

**Comment by imobile** [comment]
hi @cramforce would you please have our repo reviewed? Thanks a lot.

**Comment by imobile** [comment]
hi @ilovelili could you see me

File: ads/_config.js [hunk 1]
```diff
@@ -34,6 +34,11 @@ export const adPrefetch = {
   yieldmo: 'https://static.yieldmo.com/ym.amp1.js',
   revcontent: 'https://labs-cdn.revcontent.com/build/amphtml/revcontent.amp.min.js',
   teads: 'https://cdn.teads.tv/media/format/v3/teads-format.min.js',
+  imobile: [
+    'https://spamp.i-mobile.co.jp/script/amp.js',
```
Comment (cramforce) [inline ads/_config.js]: If you load all of these, why not just load one file?

File: ads/_config.js [hunk 2]
```diff
@@ -80,6 +85,10 @@ export const adPreconnect = {
     'https://cdn.revcontent.com',
     'https://img.revcontent.com',
   ],
+  imobile: [
+    'https://spad.i-mobile.co.jp',
```
Comment (cramforce) [inline ads/_config.js]: No need to repeat hosts mentioned above.

File: ads/imobile.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+import {writeScript} from '../src/3p';
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function imobile(global, data) {
+  	global.imobileParam = data;
```
Comment (cramforce) [inline ads/imobile.js]: nit: indent only 2 spaces

**Comment by cramforce** [comment]
Looks good. Some small comments. Please rebase against master and squash commits.

**Comment by imobile** [comment]
@cramforce Thank you very much for your quick reply. We have fixed the problems issued by you and squashed the commits.  Looking forward to hearing from your feedback soon, thanks again.

**Comment by cramforce** [comment]
Looks great. This is ready to merge, but please rebase against master and repush. There are currently merge conflicts. I believe one of the files you edited was deleted.

**Comment by cramforce** [comment]
Please squash commits.

**Comment by imobile** [comment]
@cramforce  Sorry we messed something up. We will create a new repo and resend out pull request ASAP. Sorry for the inconvenience.

**Pull request closed without merging**